io/xmlparser/CMakeLists.txt

```diff
@@ -14,15 +14,13 @@ ROOT_STANDARD_LIBRARY_PACKAGE(XMLParser
                               HEADERS
                                  TDOMParser.h
                                  TSAXParser.h
-                                 TSAXParser.h
```

Comment:
There is a typo in the commit message, "dublicated".

Reply:
fixing :)